app/services/suggestions_service.rb

```diff
@@ -11,6 +11,7 @@ def call
       ].flatten.compact
     end
     SearchSuggestion.restrict_primary_key
+    Rails.logger.debug("Query count: #{::SequelRails::Railties::LogSubscriber.count}")
```

Comment:
FYI this doesn't really work - because nothing is resetting the counter?

In the context of a call from a controller the counter gets reset in a before_action, and written out to the log in an after action - but here it will

1) When called from a controller: Show the count of all queries from the start of the controller action up to the point this line is executed

2) When it is called outside of a controller, it will probably list all queries from app boot until this line is called

_and_ you don't want to be resetting it here because it will break the query counts on countroller actions?